### DISCUSSION TITLE
Download of windows executable via official micropython website or micropython github

### DISCUSSION THREAD

**Discussion by n19k6** [micropython/micropython#11436] [Unix & Windows] [answered]
Hello,
thanks for providing the community-supported Windows port of MicroPython. I am aware that generally you would use a python distribution available via python.org. I am a hobbyist using python as well as micropython for educational projects. It is amazing for me that you can run it on microcontrolers as for example the raspberry pico.
Yesterday a friend ask me to write a small script to do a very basic task for a windows pc what could be solved e.g. by a batch script, python, perl or alike (file handling, random numbers, waits). I just started googling and writing down the batch script, but than recalled that I compiled a micropython win32 executable on my rasperry pi last year.
I recalled that from then that there had been discussions to provide the executable for download from the micropython website.
I checked today in github the current information about the windows port. I got aware of the ports/windows directory and using a debian vm on my esxi server I could build the windows executable micropython.exe within less than five minutes using the mentioned three or four commands mentioned in the README.md. For me that approach was perfect.
Since the goal of micropython is running python on microcontrollers and not on windows, and that the windows port is experimental and community based I understand that a windows executable is not available via micropython/downloads (I tried ?port=win32, ?port=windows). However I suppose there is a running pipeline creating on micropython github creating continually windows micropython executables.
My question is can they be downloaded from github or the micropython website? I am fully aware that they are experimental.

Thanks, n19k6

**Comment by stinos** [comment, marked as answer]
> However I suppose there is a running pipeline creating on micropython github creating continually windows micropython executables.

Yes there are builds both with Github and Appveyor but none of those get published. As far as I'm aware the website nor github have dowloads currently. There were PRs like https://github.com/micropython/micropython/pull/6055 to make this available but that was left unattended. I do seem to recall someone has 'unofficial' builds available but can't remember where.

**Reply by n19k6** [reply]
Thank your for the answer. Best regards, n19k6
